client_test.go

```diff
@@ -604,6 +604,39 @@ func TestGocloak_GetToken(t *testing.T) {
 	assert.NotEmpty(t, newToken.IDToken, "Got an empty if token")
 }
 
+func TestGocloak_GetRequestingPartyToken(t *testing.T) {
+	t.Parallel()
+	cfg := GetConfig(t)
+	client := NewClientWithDebug(t)
+	SetUpTestUser(t, client)
+	newToken, err := client.GetToken(
+		cfg.GoCloak.Realm,
+		TokenOptions{
+			ClientID:      &cfg.GoCloak.ClientID,
+			ClientSecret:  &cfg.GoCloak.ClientSecret,
+			Username:      &cfg.GoCloak.UserName,
+			Password:      &cfg.GoCloak.Password,
+			GrantType:     StringP("password"),
+			ResponseTypes: []string{"token", "id_token"},
+			Scopes:        []string{"openid", "offline_access"},
```

Comment:
Do you really need an offline token for GetRequestingPartyToken?

Reply:
no, I just copied the `TokenOptions` from the `GetToken` test. I can remove that if you'd like.

Reply:
yes, remove please